### PR TITLE
v 1.0.0 documentation

### PR DESCRIPTION
**Pull request by azimov**
Creating PR for @anthonysena for comments

### CODE REVIEW

File: vignettes/CreatingAnalysisSpecification.Rmd [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+
+# Creating an analysis specification
+
+This walk through will show how to use `Strategus` to define an analysis specification on an example study using cohorts from the example problem _What is the risk of gastrointestinal (GI) bleed in new users of celecoxib compared to new users of diclofenac?_ as described in the [Book Of OHDSI Chapter 12 on Population Level Estimation]( https://ohdsi.github.io/TheBookOfOhdsi/PopulationLevelEstimation.html#PopulationLevelEstimation)
+
+## Setting up your R environment
+
+Use `renv` and the [renv.lock](https://github.com/ohdsi-studies/StrategusStudyRepoTemplate/blob/main/renv.lock) from the Strategus study template to set up your R environment. This is done by copying the `renv.lock` file into the root of a new project and the restore of the environment is done by calling [renv::restore()](https://rstudio.github.io/renv/reference/restore.html). This will ensure that you have all of the R dependencies including the OHDSI HADES libraries and Strategus.
+
```
Comment (azimov) [inline vignettes/CreatingAnalysisSpecification.Rmd:42]: Could add a section:

```{r eval=FALSE}
download.file("https://github.com/ohdsi-studies/StrategusStudyRepoTemplate/blob/main/renv.lock")
install.packages("renv")
renv::restore()
```

**Review by azimov — COMMENTED** [review]
No major comments, just a few points about how we could make it more user friendly

File: vignettes/CreatingAnalysisSpecification.Rmd [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+sccsModule <- SelfControlledCaseSeriesModule$new()
+library(SelfControlledCaseSeries)
+
+# Exposures-outcomes -----------------------------------------------------------
+negativeControlOutcomeIds <- ncoCohortSet$cohortId
+outcomeOfInterestIds <- c(3)
+exposureOfInterestIds <- c(1, 2)
+
+exposuresOutcomeList <- list()
+for (exposureOfInterestId in exposureOfInterestIds) {
```
[the remark this answers is not in the document]
Comment (anthonysena) [reply vignettes/CreatingAnalysisSpecification.Rmd:284]: Agreed - I've noted this in #173 and address this in a point release.

File: vignettes/IntroductionToStrategus.Rmd
[the remark this answers is not in the document]
Comment (anthonysena) [reply vignettes/IntroductionToStrategus.Rmd:1]: Agreed - will do this or update this in the `_pkgdown.yml` configuration.

**Pull request merged**